lib/src/visitor/serialize.dart

```diff
@@ -611,7 +611,8 @@ class _SerializeVisitor
     _writeBetween<Value>(
         _inspect
             ? value.asList
-            : value.asList.where((element) => !element.isBlank),
+            : value.asList
+                .where((element) => element.hasBrackets || !element.isBlank),
```

Comment:
is this the right fix ? Or should bracketed lists not be considered blank ?

Side note: the commit message says this is for `inspect`, but the fix is for the CSS serialization, not the inspect one.